rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy

```diff
@@ -1042,8 +1042,18 @@ class ExecutionService implements ApplicationContextAware, StepExecutor, NodeSte
 //
 
         //set up log output threshold
-        def thresholdMap = ScheduledExecution.parseLogOutputThreshold(scheduledExecution?.logOutputThreshold)
-        def threshold = LoggingThreshold.fromMap(thresholdMap,scheduledExecution?.logOutputThresholdAction)
+        boolean logGlobalConfig = frameworkService.getProjectLogGlobalConfig()
+
+        boolean limitFramework = false
```

Comment:
the `limitFramework` variable will not work the way I think you are trying to use it.  Please write some tests.

Reply:
Done... variable removed